get_the_books.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+book_ids=(17192 32037 932 1064 10947 1063 51060 2147 2151 2150 2148)
+
+for id in "${book_ids[@]}"
+do
+    wget "https://www.gutenberg.org/cache/epub/$id/pg$id.txt" -O "book_$id.txt"
```

Comment:
It's not an issue here, but I've gotten into the habit of using {} around a variable to avoid bash getting confused.  In other words `... pub/${id}/pg${id}.txt` prevents bash from looking for a variable say named `id.txt`